src/main/java/com/openshift/restclient/model/IReplicationController.java

```diff
@@ -70,7 +71,7 @@
 	 * @param containerPorts  the container ports
 	 * @param volumes		 the set of emptyDir volumes to add to the config
 	 */
-	void addContainer(String name, DockerImageURI tag,  Set<IPort> containerPorts, Map<String, String> envVars, List<String> volumes);
+	IContainer addContainer(String name, DockerImageURI tag,  Set<IPort> containerPorts, Map<String, String> envVars, List<String> volumes);
```

Comment:
3.1 was released as SNAPSHOT only though, so I would agree in saying, let's do it